js/main.js

```diff
@@ -0,0 +1,45 @@
+//Happy Birthday Leader! 🎇💙
+
+const card_list = document.getElementById("card-list");
+const CLOUD_NAME = "dazcxdgiy";
+const CLOUDINARY_URL = `https://res.cloudinary.com/${CLOUD_NAME}/image/upload/`;
+
+//Custom Card component. Use it like this:
+//<tcg-card card-id="[CARD_ID]"></tcg-card>
+class Card extends HTMLElement {
+  constructor() {
+    super();
+    const image = document.createElement("img");
+    image.src = this.getImageURL();
+    image.classList.add("card-image");
+    this.appendChild(image);
+  }
+
+  //This returns an url of the form:
+  //`https://res.cloudinary.com/${CLOUD_NAME}/image/upload/cld-sample-${CARD_ID}`
+  //if "card-id" isn't supplied it returns a F L O W E R 🌸 sample image url.
+  getImageURL() {
+    return `${CLOUDINARY_URL}${
```

Comment:
You can do this in a different PR, but is there a way we can also get a given image's metadata? I'm thinking about how we can fetch a card's text description, rarity, etc.

Reply:
How did Cloudinary get the alt text etc? Did you set that in the Cloudinary UI?

Reply:
Yep. You can open the media library, open an image, go to the metadata tab and set everything there.